baselines/dom.generated.d.ts

```diff
@@ -18550,6 +18553,12 @@ type AudioContextLatencyCategory = "balanced" | "interactive" | "playback";
 type AudioContextState = "closed" | "running" | "suspended";
 type AuthenticatorAttachment = "cross-platform" | "platform";
 type AuthenticatorTransport = "ble" | "hybrid" | "internal" | "nfc" | "usb";
+type AutoFillAddressKind = "billing" | "shipping";
+type AutoFillBase = "off" | "on";
+type AutoFillContactField = "email" | "impp" | "tel" | "tel-area-code" | "tel-country-code" | "tel-extension" | "tel-local" | "tel-local-prefix" | "tel-local-suffix" | "tel-national";
+type AutoFillContactKind = "fax" | "home" | "mobile" | "pager" | "work";
+type AutoFillCredentialField = "webauthn";
```

Comment:
Not yet supported in Gecko but supported in all other engines, and critical for Passkeys support; kept.